tests/config/tags.yml

```diff
@@ -128,14 +128,25 @@ markduplicates:
   - conf/modules/markduplicates.config
   - modules/nf-core/gatk4/markduplicates/main.nf
   - modules/nf-core/mosdepth/main.nf
-  - modules/nf-core/samtools/convert/main.nf
```

Comment:
why?

Reply:
Don't remember. Probably a mistake

Reply:
Yes, I'll do that.

Reply:
oh, now that I get to think of it, I removed it because it seems to me that samtools/convert isn't used by the markduplicates-subworkflow.

Reply:
ok, I'll have a look then, but please restore it in this PR

Reply:
it belonged there: https://github.com/nf-core/sarek/blob/dev/workflows/sarek.nf#L588